### DISCUSSION TITLE
Rustdesk connection through a Unifi USG network

### DISCUSSION THREAD

**Discussion by liamf91** [rustdesk/rustdesk#7212] [General]
Good day,

I am having some difficulty establishing a connection with the standard Rustdesk servers. I have troubleshooted to a point where I know its not the endpoint (laptops). 

I have a Unifi network, which recently I have to factory reset due to faulty hardware needing to be replaced (unfortunately the backups I had didn't restore everything in terms of settings). Before the reset there must have been a setting which was enable/disabled/allowed etc, which I was able to connect to Rustdesk. I am able to connect when I use my phone as a hotspot WiFi.

Now that the settings are back to how they were, its not allowing me to connect to the Rustdesk servers.

I have allowed ports 21115-21119 in the firewall for any protocol and any device (security wise naughty I know). how ever this didnt help.

Has anyone else had a similar issue? 

Please note, this is the free edition, not the Pro (Self-host) edition.

**Comment by DMSnz** [comment]
Might need more details to help but simple ideas;
are your laptops on wireless with guest portal settings enabled (segregation for example) that might block wifi to RustDesk server network?
Other settings there were restored, can not speak for backup/restore but I have seen after a firmware update, settings say one thing and do another, requiring a fake change and change back / save to fix.
We have extensive (for now) Unifi networking, UXG Pro routers + USGs, VPNs etc and across all sites RustDesk works fine.
But ours is typically accessible via VPN only, but I did recently open ports on the firewall to allow a connection that had no VPN  and this just worked without issue.
Those ports for reference were;
21114, 21115, 21116, 21118 to the main RustDesk services
21117 and 21119 for relay.

**Reply by liamf91** [reply]
Thank you. It was probably user error. Previously, I had opened the ports above on the network, but this didn't have any affect. So I disabled that change. Yesterday I reenabled the ports, seemed to work this time. (typically my luck)

Thank you for your help.
